modal.css

```diff
@@ -0,0 +1,158 @@
+.subtittle {
+  margin-right: 2%;
+  float: left;
+}
+
+h2.popUp {
+  margin-top: 10%;
+  margin-bottom: 10%;
+  width: 50;
+}
+
+.popupContainer {
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 95%;
+  z-index: 2;
+  padding-left: 0.5rem;
+  padding-right: 0.5rem;
+  background: #fff;
+  justify-content: start;
+  margin-right: 3%;
+  margin-left: 3%;
+  margin-top: 4%;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
- [ ] The mobile popup could also use some adjustment as I've noticed the current height you've specified is filling up almost the entire screen so please make sure the height is similar as specified in the Figma design🎨:
- Your design:
  ![image](https://user-images.githubusercontent.com/104968236/233393609-a31beae6-7109-465d-be1f-186f48ff1cf7.png)
- Figma design:
  ![image](https://user-images.githubusercontent.com/104968236/233393744-b480b2e6-dcc7-4765-9e35-96c70efe40f2.png)
- As you can see from the above Figma design the height doesn't take up the entire screen.